tests/ext/test_application.py

```diff
@@ -1581,7 +1581,13 @@ def thread_target():
         async def post_init(app: Application) -> None:
             events.append("post_init")
 
-        app = Application.builder().bot(one_time_bot).post_init(post_init).build()
+        app = (
+            Application.builder()
+            .application_class(PytestApplication)
+            .updater(PytestUpdater(one_time_bot, asyncio.Queue()))
```

Comment:
ah, you use `PytestApplication` in more places, too. What's the reason behind this?

Reply:
since the slots for application now actually don't have a `__dict__`, we can get an error like:

``` python
>       monkeypatch.setattr(
            app,
            "stop",
            call_after(app.stop, lambda _: events.append("app.stop")),
        )
E       AttributeError: 'Application' object attribute 'stop' is read-only
```
so we need PytestApplication to actually be able to monkeypatch again